.ci_scripts/generate_governance_members.py

```diff
@@ -0,0 +1,35 @@
+import csv
+from pathlib import Path
+
+
+REPO_DIR = Path(__file__).parents[1].absolute()
+GOVERNANCE_MD = REPO_DIR / "community" / "governance.md"
```

Comment:
I wonder if we should have `governance.md.tmpl` which is then rendered to `.md`. Or, otherwise, something native to JSX so we can fetch the CSVs at build time. Maybe that's more difficult to maintain, though.

I know we have a precedent with the CFEP list, just leaving some thoughs in the air.

Reply:
I have an idea to render this using native JSX indeed.

Reply:
It took a bit longerr than I wanted but added a small plugin to update webpack config to be able to import csv files and convert them to json to then be able to render them using a small custom react component 🙃 

Yeiii?